src/main/java/com/neighborhoodAssurance/NAController.java

```diff
@@ -23,6 +23,15 @@ public ModelAndView home() {
 		modelAndView.addObject("agencyDTO", agencyDTO);
 		return modelAndView;
 	}
+	/**
+	 * Endpoint for a user to log in
+	 * Adds the ability to create personolized information
+	 * @return
+	 */
+	@RequestMapping(value="/login", method=RequestMethod.GET)
+	public String login{
+		return "login";
+	}

```

Comment:
Is there a corresponding login.html page?  I don't see one in this commit.  If not, this will cause an error.